apps/aqua/src/index.ts

```diff
@@ -22,41 +24,60 @@ app.get("/client-metadata.json", (c) => {
 });
 
 app.get("/", async (c) => {
-  const cookies = getCookie(c, "tealSession");
-  const sessCookie = cookies?.split("teal:")[1];
+  const tealSession = getCookie(c, "tealSession");
 
   // Serve logged in content
-  if (sessCookie != undefined) {
-    const session = await getContextDID(c);
-
-    if (session != undefined) {
-      const agent = await getSessionAgent(c);
-      const post = await agent?.getPost({repo: "teal.fm", rkey: "3lb2c74v73c2a"});
-      // const agent = await getSessionAgent(c);
-      // const followers = await agent?.getFollowers();
-      return c.html(
-        `<div id="root">
-          <div id="header">
+  if (tealSession) {
+    const agent = await getSessionAgent(c);
+
+    const post = await agent?.getPost({
+      repo: "teal.fm",
+      rkey: "3lb2c74v73c2a",
+    });
+    // const followers = await agent?.getFollowers();
+    return c.html(
+      `
+      <head>
+      <link rel="stylesheet" href="https://latex.vercel.app/style.css">
```

Comment:
can we serve this stylesheet ourselves orrrr

Reply:
something like this should work for serving the css and fonts for now https://hono.dev/docs/getting-started/nodejs#serve-static-files